plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectConfig.java

```diff
@@ -26,64 +26,64 @@
 import org.eclipse.core.resources.IProject;
 
 /**
- * Collects all data needed to create and configure an App Engine Standard Project.
+ * Collects all data needed to create and configure an App Engine Project.
  */
-class AppEngineStandardProjectConfig {
+public class AppEngineProjectConfig {
```

Comment:
I start to get nervous when methods get more public. This is now exposing mutable internal state publicly.